libparsec/crates/platform_mountpoint/src/windows/mod.rs

```diff
@@ -510,3 +508,18 @@ pub fn mount<T: MountpointInterface>(
         .map(|fs| FileSystemMounted { fs })
         .map_err(|status| anyhow::anyhow!("Failed to init FileSystem {status}"))
 }
+
+fn os_path_to_fs_path(path: &U16CStr) -> Result<FsPath, NTSTATUS> {
+    path.to_os_string()
+        .to_str()
+        .map(|s| {
+            s.replace('\\', "/")
+                .split('/')
```

Comment:
```suggestion
            s.split('\\')
```

directly split by `\` ?

Reply:
If your `path` contains '/' instead of `\`, it will fails, there you aggregate, it's not the same thing

Reply:
I agree that currently `winfsp` will returns path only with `\`, but it's not my job to know precisely which separator the `OS` will use